docs/best_practices/package_xml.rst

```diff
@@ -0,0 +1,40 @@
+Packages
+========
+
+Before you start writing a new tool please search the `Main Tool Shed
+(MTS) <https://toolshed.g2.bx.psu.edu>`__ and the `Test Tool Shed
+(TTS) <https://testtoolshed.g2.bx.psu.edu>`__ because it's possible that
+someone has already created a wrapper for the same third party
+executable you are looking for. Consider announcing your tool project on
+galaxy-dev to see if anyone has already created a wrapper.
+
+Packaging software is something of a more advanced topic, and due to the
+complexities of the syntax, somewhat harder to validate.
+
+
+Downloads
+---------
+
+Packages generally must download one or more files from the internet in order
+to function. We require checksums on all of our package downloads from multiple reasons:
+
+-  Download integrity.
+-  Insecure transport methods, like ``http://`` and ``ftp://``
+-  The packages come from the untrusted internet, we don't know if anyone has
+   modified the software in transit. This software is installed directly to
+   large university clusters. We must make an effort to ensure that what is
+   being installed is what the user actually asked for, and not a version of
+   ``bowtie`` that the NSA has put together just for *you*.
```

Comment:
It's PC in that it won't really offend anyone, because everyone knows this stuff is happening.. 

We can change it to something less politically charged, but it was just a bit of colour in otherwise pretty dull documentation.